src/pybind/mgr/dashboard/ci/cephadm/bootstrap-cluster.sh

```diff
@@ -12,6 +12,26 @@ CEPHADM_SRC="/mnt/{{ ceph_dev_folder }}/src/cephadm/cephadm"
 
 cp $CEPHADM_SRC $CEPHADM
 
+# default configs for the cluster to make it
+# behave like a single node cluster
+declare -A ceph_configs=(
+  ["mon_allow_pool_size_one"]="true"
+  ["mon_allow_pool_delete"]="true"
+  ["osd_pool_default_min_size"]="1"
+  ["osd_pool_default_size"]="1"
+  ["mon_data_avail_crit"]="1"
+  ["mon_data_avail_warn"]="1"
+)
+
+declare -A config_sections=(
+  ["mon_allow_pool_size_one"]="mon"
+  ["mon_allow_pool_delete"]="mon"
+  ["osd_pool_default_min_size"]="global"
+  ["osd_pool_default_size"]="global"
+  ["mon_data_avail_crit"]="mon"
+  ["mon_data_avail_warn"]="mon"
+)
```

Comment:
`cephadm bootstrap` supports a `--config <config.file>` to import a basic config file. I'd rather add that as an additional file and copy to the cluster through the `- files` YAML key:
```ini
[global]
osd_pool_default_min_size=1
osd_pool_default_size=1
...
```

When things look overcomplicated, probably there's a simpler way to do it.

Reply:
done here